### DISCUSSION TITLE
AI in a Nutshell – ForBo7 // Salman Naqvi

### DISCUSSION THREAD

**Discussion by giscus** [ForBo7/forbo7.github.io#15] [Giscus Comments]
# AI in a Nutshell – ForBo7 // Salman Naqvi

AI models are much, much simpler than you think.

https://forbo7.github.io/forblog/posts/6_ai_in_a_nutshell.html

**Comment by CraigRichards** [comment]
Thanks for sharing

**Reply by ForBo7** [reply]
If you have any questions or confusions, do let me know! 😀

**Comment by sevigti** [comment]
Great explanation, well done.

**Reply by ForBo7** [reply]
I appreciate the comment! 😄

If you think there's any way I could improve this post, as it was written quite a while back when I was new to all this, do let me know!
